packages/api/lib/errors.js

```diff
@@ -8,7 +8,11 @@ const {
 } = require('@cumulus/errors');
 
 const isBadRequestError = (err) =>
-  err.name === 'SchemaValidationError' || err instanceof ValidationError;
+  err.name === 'SchemaValidationError'
+  || err instanceof ValidationError
+  || ['22', '23'].includes((err.code || '').substring(0, 2));
```

Comment:
Essentially all the 22xxx and 23xxx errors from postgres are data/integrity violations which I believe fit the bill for the same class of error.

Reply:
Consider abstracting postgres error eval to descriptive function

Reply:
1822429ce